.github/workflows/deploy.yml

```
@@ -163,6 +163,137 @@ jobs:
           GITHUB_TOKEN: ${{ github.token }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
 
+  android_hybrid:
+    name: Build and deploy Android HybridApp
+    needs: prep
+    runs-on: ubuntu-latest-xl
+    # Only deploy HybridApp to staging
+    if: ${{ github.ref != 'refs/heads/production' }}
```

NAB but this ensures we're indeed deploying only staging
```suggestion
    if: ${{ github.ref == 'refs/heads/staging' }}
```